pkg/prometheus/promcfg_test.go

```diff
@@ -1584,6 +1584,7 @@ func TestNoEnforcedNamespaceLabelServiceMonitor(t *testing.T) {
 					Namespace: "default",
 				},
 				Spec: monitoringv1.ServiceMonitorSpec{
+					SelectorMechanism: ptr.To(monitoringv1.SelectorMechanismRelabel),
```

Comment:
do we need to define the SelectorMechanism value?

Reply:
Not really, removed